feat: add check before syncing doc

yqrashawn opened the pull request:
fixes #20470

### Summary

This PR relies on new onboarding flow in PR #20386
Will be ready for review once that PR merged

### Review notes
add `status-im.common.check-before-syncing.view`
`status-im.common.check-before-syncing.view/view` can be used by other code in combination with `:show-bottom-sheet`


### Testing notes
flow changed, affects e2e tests

#### Platforms
<!-- (Optional. Specify which platforms should be tested) -->

- Android
- iOS

#### Areas that maybe impacted
- sign in by syncing after intro
- sign in by syncing after logged out
- add syncing device after logged in

status: ready <!-- Can be ready or wip -->

yqrashawn committed:
feat: check before syncing doc

Signed-off-by: yqrashawn <namy.19@gmail.com>

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (20)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 98991dcd656298d67e455cf5aab8121b611d8619 | [#1](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-20504/1/) | 2024-06-19 02:14:10 | ~4 min | `tests` | [:page_facing_up:`log`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-240619-020913-98991dcd656298d67e455cf5aab8121b611d8619-pr20504-tests.log) |
| :heavy_check_mark: | 98991dcd656298d67e455cf5aab8121b611d8619 | [#1](https://ci.status.im/job/status-mobile/job/prs/job/android-e2e/job/PR-20504/1/) | 2024-06-19 02:17:51 | ~8 min | `android-e2e` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-240619-020908-98991dcd656298d67e455cf5aab8121b611d8619-pr20504-x86_64.apk) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-240619-020908-98991dcd656298d67e455cf5aab8121b611d8619-pr20504-x86_64.apk)|
| :heavy_check_mark: | 98991dcd656298d67e455cf5aab8121b611d8619 | [#1](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-20504/1/) | 2024-06-19 02:18:02 | ~8 min | `android` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-240619-020908-98991dcd656298d67e455cf5aab8121b611d8619-pr20504-universal.apk) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-240619-020908-98991dcd656298d67e455cf5aab8121b611d8619-pr20504-universal.apk)|
| :heavy_check_mark: | 98991dcd656298d67e455cf5aab8121b611d8619 | [#1](https://ci.status.im/job/status-mobile/job/prs/job/ios/job/PR-20504/1/) | 2024-06-19 02:19:13 | ~10 min | `ios` | [:iphone:`ipa`](https://i.diawi.com/agb1ya) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fi.diawi.com%2Fagb1ya)|
| | | | | | | |
| :heavy_check_mark: | d21019ba5d8ba041619e953ca4f4596f00be55a7 | [#2](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-20504/2/) | 2024-06-19 02:26:20 | ~4 min | `tests` | [:page_facing_up:`log`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-240619-022212-d21019ba5d8ba041619e953ca4f4596f00be55a7-pr20504-tests.log) |
| :heavy_check_mark: | d21019ba5d8ba041619e953ca4f4596f00be55a7 | [#2](https://ci.status.im/job/status-mobile/job/prs/job/android-e2e/job/PR-20504/2/) | 2024-06-19 02:30:38 | ~8 min | `android-e2e` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-240619-022209-d21019ba5d8ba041619e953ca4f4596f00be55a7-pr20504-x86_64.apk) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-240619-022209-d21019ba5d8ba041619e953ca4f4596f00be55a7-pr20504-x86_64.apk)|
| :heavy_check_mark: | d21019ba5d8ba041619e953ca4f4596f00be55a7 | [#2](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-20504/2/) | 2024-06-19 02:30:40 | ~8 min | `android` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-240619-022209-d21019ba5d8ba041619e953ca4f4596f00be55a7-pr20504-universal.apk) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-240619-022209-d21019ba5d8ba041619e953ca4f4596f00be55a7-pr20504-universal.apk)|
| :heavy_check_mark: | d21019ba5d8ba041619e953ca4f4596f00be55a7 | [#2](https://ci.status.im/job/status-mobile/job/prs/job/ios/job/PR-20504/2/) | 2024-06-19 02:31:50 | ~9 min | `ios` | [:iphone:`ipa`](https://i.diawi.com/4gDeKm) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fi.diawi.com%2F4gDeKm)|
| | | | | | | |
| :heavy_check_mark: | a627bdd0cb807dabc5c313c9541c6f62bfb89f85 | [#3](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-20504/3/) | 2024-06-25 03:10:39 | ~5 min | `tests` | [:page_facing_up:`log`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-240625-030531-a627bdd0cb807dabc5c313c9541c6f62bfb89f85-pr20504-tests.log) |
| :heavy_check_mark: | a627bdd0cb807dabc5c313c9541c6f62bfb89f85 | [#3](https://ci.status.im/job/status-mobile/job/prs/job/android-e2e/job/PR-20504/3/) | 2024-06-25 03:11:53 | ~6 min | `android-e2e` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-240625-030531-a627bdd0cb807dabc5c313c9541c6f62bfb89f85-pr20504-x86_64.apk) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-240625-030531-a627bdd0cb807dabc5c313c9541c6f62bfb89f85-pr20504-x86_64.apk)|
| :heavy_check_mark: | a627bdd0cb807dabc5c313c9541c6f62bfb89f85 | [#3](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-20504/3/) | 2024-06-25 03:14:01 | ~8 min | `android` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-240625-030531-a627bdd0cb807dabc5c313c9541c6f62bfb89f85-pr20504-universal.apk) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-240625-030531-a627bdd0cb807dabc5c313c9541c6f62bfb89f85-pr20504-universal.apk)|
| :heavy_check_mark: | a627bdd0cb807dabc5c313c9541c6f62bfb89f85 | [#3](https://ci.status.im/job/status-mobile/job/prs/job/ios/job/PR-20504/3/) | 2024-06-25 03:15:21 | ~9 min | `ios` | [:iphone:`ipa`](https://i.diawi.com/yKSQ5i) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fi.diawi.com%2FyKSQ5i)|
| | | | | | | |
| :heavy_check_mark: | 3455dbfbbe7e8184ab7e10c3ec5d594855490977 | [#4](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-20504/4/) | 2024-06-26 03:18:18 | ~4 min | `tests` | [:page_facing_up:`log`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-240626-031402-3455dbfbbe7e8184ab7e10c3ec5d594855490977-pr20504-tests.log) |
| :heavy_check_mark: | 3455dbfbbe7e8184ab7e10c3ec5d594855490977 | [#4](https://ci.status.im/job/status-mobile/job/prs/job/android-e2e/job/PR-20504/4/) | 2024-06-26 03:20:05 | ~6 min | `android-e2e` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-240626-031402-3455dbfbbe7e8184ab7e10c3ec5d594855490977-pr20504-x86_64.apk) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-240626-031402-3455dbfbbe7e8184ab7e10c3ec5d594855490977-pr20504-x86_64.apk)|
| :heavy_check_mark: | 3455dbfbbe7e8184ab7e10c3ec5d594855490977 | [#4](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-20504/4/) | 2024-06-26 03:21:27 | ~7 min | `android` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-240626-031402-3455dbfbbe7e8184ab7e10c3ec5d594855490977-pr20504-universal.apk) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-240626-031402-3455dbfbbe7e8184ab7e10c3ec5d594855490977-pr20504-universal.apk)|
| :heavy_check_mark: | 3455dbfbbe7e8184ab7e10c3ec5d594855490977 | [#4](https://ci.status.im/job/status-mobile/job/prs/job/ios/job/PR-20504/4/) | 2024-06-26 03:23:56 | ~9 min | `ios` | [:iphone:`ipa`](https://i.diawi.com/4RFMwk) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fi.diawi.com%2F4RFMwk)|
| | | | | | | |
| :heavy_check_mark: | 4164d43d2c6f4ebdba1d599cb2a6991a942aaaaa | [#5](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-20504/5/) | 2024-06-27 01:17:37 | ~4 min | `tests` | [:page_facing_up:`log`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-240627-011256-4164d43d2c6f4ebdba1d599cb2a6991a942aaaaa-pr20504-tests.log) |
| :heavy_check_mark: | 4164d43d2c6f4ebdba1d599cb2a6991a942aaaaa | [#5](https://ci.status.im/job/status-mobile/job/prs/job/android-e2e/job/PR-20504/5/) | 2024-06-27 01:21:34 | ~8 min | `android-e2e` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-240627-011252-4164d43d2c6f4ebdba1d599cb2a6991a942aaaaa-pr20504-x86_64.apk) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-240627-011252-4164d43d2c6f4ebdba1d599cb2a6991a942aaaaa-pr20504-x86_64.apk)|
| :heavy_check_mark: | 4164d43d2c6f4ebdba1d599cb2a6991a942aaaaa | [#5](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-20504/5/) | 2024-06-27 01:21:36 | ~8 min | `android` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-240627-011252-4164d43d2c6f4ebdba1d599cb2a6991a942aaaaa-pr20504-universal.apk) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-240627-011252-4164d43d2c6f4ebdba1d599cb2a6991a942aaaaa-pr20504-universal.apk)|
| :heavy_check_mark: | 4164d43d2c6f4ebdba1d599cb2a6991a942aaaaa | [#5](https://ci.status.im/job/status-mobile/job/prs/job/ios/job/PR-20504/5/) | 2024-06-27 01:22:41 | ~9 min | `ios` | [:iphone:`ipa`](https://i.diawi.com/2QVWQw) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fi.diawi.com%2F2QVWQw)|
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | fb0e26cebf472f66a30661e5bbed3374cb72e119 | [#6](https://ci.status.im/job/status-mobile/job/prs/job/android-e2e/job/PR-20504/6/) | 2024-07-01 01:58:34 | ~8 min | `android-e2e` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-240701-015013-fb0e26cebf472f66a30661e5bbed3374cb72e119-pr20504-x86_64.apk) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-240701-015013-fb0e26cebf472f66a30661e5bbed3374cb72e119-pr20504-x86_64.apk)|
| :heavy_check_mark: | fb0e26cebf472f66a30661e5bbed3374cb72e119 | [#6](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-20504/6/) | 2024-07-01 01:58:38 | ~8 min | `android` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-240701-015013-fb0e26cebf472f66a30661e5bbed3374cb72e119-pr20504-universal.apk) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-240701-015013-fb0e26cebf472f66a30661e5bbed3374cb72e119-pr20504-universal.apk)|
| :heavy_check_mark: | fb0e26cebf472f66a30661e5bbed3374cb72e119 | [#6](https://ci.status.im/job/status-mobile/job/prs/job/ios/job/PR-20504/6/) | 2024-07-01 02:00:30 | ~10 min | `ios` | [:iphone:`ipa`](https://i.diawi.com/HCNCES) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fi.diawi.com%2FHCNCES)|
| | | | | | | |
| :heavy_check_mark: | b1859b30 | [#7](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-20504/7/) | 2024-07-02 02:37:46 | ~4 min | `tests` | [:page_facing_up:`log`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-240702-023250-b1859b-pr20504-tests.log) |
| :heavy_check_mark: | b1859b30 | [#7](https://ci.status.im/job/status-mobile/job/prs/job/android-e2e/job/PR-20504/7/) | 2024-07-02 02:41:27 | ~8 min | `android-e2e` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-240702-023245-b1859b-pr20504-x86_64.apk) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-240702-023245-b1859b-pr20504-x86_64.apk)|
| :heavy_check_mark: | b1859b30 | [#7](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-20504/7/) | 2024-07-02 02:41:28 | ~8 min | `android` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-240702-023245-b1859b-pr20504-universal.apk) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-240702-023245-b1859b-pr20504-universal.apk)|
| :heavy_check_mark: | b1859b30 | [#7](https://ci.status.im/job/status-mobile/job/prs/job/ios/job/PR-20504/7/) | 2024-07-02 02:42:37 | ~9 min | `ios` | [:iphone:`ipa`](https://i.diawi.com/bY1DLz) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fi.diawi.com%2FbY1DLz)|

yqrashawn commented:
https://github.com/status-im/status-mobile/assets/15090582/969aba67-5461-4592-a86b-2ee8a9c3c09a


https://github.com/status-im/status-mobile/assets/15090582/fa3d459a-54a0-4d92-8f09-b100a86bbe82

Parveshdhull commented on src/status_im/common/check_before_syncing/style.cljs:
```diff
@@ -0,0 +1,18 @@
+
+(def button {:flex 1})
+
+(def checkbox {:margin-top 16 :flex-direction :row :align-items :center})
+(def checkbox-text {:margin-left 10})
```
Thank you @yqrashawn for your PR.
Please format these lines. (empty lines and map)

Parveshdhull commented on src/status_im/common/check_before_syncing/view.cljs:
```diff
@@ -0,0 +1,64 @@
+    [status-im.common.check-before-syncing.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+
```
remove empty line

Parveshdhull commented on src/status_im/common/check_before_syncing/view.cljs:
```diff
@@ -0,0 +1,64 @@
+
+(defn view
+  []
+  (let [all-checked?            (reagent/atom 0)
```
Shouldn't we use rn/use-state instead?

Parveshdhull replied:
why all-checked? is a number?

Parveshdhull commented on translations/en.json:
```diff
@@ -179,6 +179,11 @@
     "chat-settings": "Chat settings",
     "chat-with-friends": "Chat privately with friends",
     "chats": "Chats",
+    "check-before-syncing":"Check before syncing",
```
Other translations also have empty space after `:`, maybe we should add for consistency.

Parveshdhull commented on src/status_im/common/check_before_syncing/view.cljs:
```diff
@@ -0,0 +1,64 @@
+                                         #(if %
+                                            (swap! all-checked? inc)
+                                            (swap! all-checked? dec)))]
+    (fn [{:keys [on-continue]}]
```
I think we will not need this.

yqrashawn commented:
Thanks for the review @Parveshdhull 
I've addressed all the above reviews

Parveshdhull commented on translations/en.json:
```diff
@@ -179,6 +179,11 @@
     "chat-with-friends": "Chat privately with friends",
     "chats": "Chats",
+    "check-before-syncing": "Check before syncing",
+    "check-before-syncing-doc-description":"To sync your devices successfully, make sure to check and complete these steps:",
+    "check-before-syncing-doc-checkbox-1":"Both devices are on the same network",
+    "check-before-syncing-doc-checkbox-2":"Make sure you are logged in on the other device",
+    "check-before-syncing-doc-checkbox-3":"Disable the firewall on your device",
```
spaces are still missing at other occurrences.

yqrashawn replied:
Uh sorry for that, I forgot that I added so many translations in this PR. I'll fix them in this PR.

clauxx commented on src/status_im/common/check_before_syncing/view.cljs:
```diff
@@ -0,0 +1,70 @@
+
+(defn- compute-new-checked-count
+  [set-checked-count checked?]
+  (if checked?
+    (set-checked-count #(inc %))
+    (set-checked-count #(dec %))))
```
could just use `(set-checked-count inc)` and same for `dec`?

yqrashawn replied:
fixed, I thought there's another arg after prevState and turns out there's not

clauxx commented on src/status_im/common/check_before_syncing/view.cljs:
```diff
@@ -0,0 +1,70 @@
+  (rf/dispatch [:hide-bottom-sheet]))
+
+(defn view
+  [{:keys [on-continue]}]
```
could rename this to `on-submit` to avoid having `real-on-continue`

clauxx reviewed (approved):
Looks good!

yqrashawn commented:
Thanks for the review @Parveshdhull @clauxx, new commit addressed above review comments

status-im-auto commented:
## 80% of end-end tests have passed
```
Total executed tests: 51
Failed tests: 6
Expected to fail tests: 4
Passed tests: 41
```
```
IDs of failed tests: 727230,702809,702777,703503,727229,702839 
```
```
IDs of expected to fail tests: 703495,702948,727232,703629 
```
<h3>Failed tests (6)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=feat/check-before-syncing&PR_ID=20504&APK_NAME=20504.apk&TR_CASE_IDS=727230,702809,702777,703503,727229,702839">Rerun failed tests</a></li><br/><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589087">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><p><blockquote><div>Device 1: Looking for a message by text: quote reply (one row) </div><div>Device 2: Looking for a message by text: quote reply (one row) </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:1009: in test_community_markdown_support
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 bold text in asterics is not displayed with markdown in 1-1 chat for the recipient (device 2)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2d2ecfd2b73a4342992a52a86aadc689?auth=7d89aadb57d19221d8d8039486c09803#188">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cc7cb86ff7534f88a2093bdd242ae785?auth=3528ca6db1fa99648dfd97179c95e8ac#167">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589121">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td><p><blockquote><div># STEP: Getting ETH amount in the wallet of the sender before transaction</div><div>Device 1: Find `WalletTab` by `accessibility id`: `wallet-stack-tab` </div></blockquote></p>

```
critical/test_wallet.py:152: in test_wallet_send_asset_from_drawer
    sender_balance, receiver_balance, eth_amount_sender, eth_amount_receiver = self._get_balances_before_tx()
critical/test_wallet.py:40: in _get_balances_before_tx
    self.wallet_1.wallet_tab.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: WalletTab by accessibility id: `wallet-stack-tab` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589120">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td><p><blockquote><div>Device 1: Swiping right on element SlideButton </div><div>Device 1: Find `SlideButton` by `xpath`: `//*[@resource-id='slide-button-track']` </div></blockquote></p>

```
critical/test_wallet.py:132: in test_wallet_send_eth
    self.wallet_1.send_asset(address=self.receiver['wallet_address'], asset_name='Ether', amount=amount_to_send)
../views/wallet_view.py:130: in send_asset
    self.confirm_transaction()
../views/wallet_view.py:118: in confirm_transaction
    self.slide_and_confirm_with_password()
../views/wallet_view.py:112: in slide_and_confirm_with_password
    self.slide_button_track.slide()
../views/base_view.py:257: in slide
    self.swipe_right_on_element(width_percentage=1.3, start_x=100)
../views/base_element.py:308: in swipe_right_on_element
    location, size = self.get_element_coordinates()
../views/base_element.py:294: in get_element_coordinates
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: SlideButton by xpath: `//*[@resource-id='slide-button-track']` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589080">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><p><blockquote><div>Device 2: Swiping right on element SlideButton </div><div>Device 2: Find `SlideButton` by `xpath`: `//*[@resource-id='slide-button-track']` </div></blockquote></p>

```
activity_center/test_activity_center.py:222: in test_add_contact_field_validation
    sync_code = self.profile_2.get_sync_code()
../views/profile_view.py:548: in get_sync_code
    self.slide_button_track.swipe_right_on_element(width_percentage=1.3)
../views/base_element.py:308: in swipe_right_on_element
    location, size = self.get_element_coordinates()
../views/base_element.py:294: in get_element_coordinates
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: SlideButton by xpath: `//*[@resource-id='slide-button-track']` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5bf06561a73a47189a71f44517b15a60?auth=d7bf57863b4b317779b585539ec26bb1#134">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/53f9cda629c34fc095144736dfd0cb37?auth=fb34da4f1fac575aba0a366a36c1c842#218">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589090">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><p><blockquote><div>Device 2: Tap on found: SendMessageButton </div><div>Device 1: Looking for a message by text: delete for me </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:401: in test_community_message_delete
    self.channel_1.chat_element_by_text(message_to_delete_for_me).wait_for_element(120)
../views/base_element.py:129: in wait_for_element
    raise TimeoutException(
 Device `1`: `ChatElementByText` by` xpath`: `//*[starts-with(@text,'delete for me')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']` is not found on the screen after wait_for_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c361512893954eaaa83b5a3543218a4b?auth=cbda886319ff0cbd4d73f8c19de2eb81#200">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cca1dbaedc234739ba0b1b574254b60d?auth=057b0a8b085052065147994da05893d2#135">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589116">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><p><blockquote><div># STEP: Check Status community tags in the Discover communities screen</div><div># STEP: Check Status Inu community tags in the Discover communities screen</div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:300: in test_community_discovery
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Community 'Status' is not in the Discover Communities list.
E    Community 'Status Inu' is not in the Discover Communities list.
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/170ec4f93b254a509084ad1016405c43?auth=85e9d732532d57b73c433edc12138290#354">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Expected to fail tests (4)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589104">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Text` by `xpath`: `//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'#cats')]` </div><div>Device 2: Looking for a message by text: it is just a message text </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:1124: in test_community_hashtag_links_to_community_channels
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Message in community channel is not visible for user before join 
```

<b>[[Can't enter channel after community is fetched for the first time, https://github.com/status-im/status-mobile/issues/20395]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2d2ecfd2b73a4342992a52a86aadc689?auth=7d89aadb57d19221d8d8039486c09803#307">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cc7cb86ff7534f88a2093bdd242ae785?auth=3528ca6db1fa99648dfd97179c95e8ac#295">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589117">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td><p><blockquote><div>Device 2: Looking for community: 'open community' </div><div>Device 2: Click until `Text` by `accessibility id`: `community-description-text` will be presented </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:1212: in test_community_join_when_node_owner_offline
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Message in community channel is not visible for user before join, it was indicated as not sent for the sender before he went offline 
```

<b>[[Can't enter channel after community is fetched for the first time, https://github.com/status-im/status-mobile/issues/20395]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2d2ecfd2b73a4342992a52a86aadc689?auth=7d89aadb57d19221d8d8039486c09803#508">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cc7cb86ff7534f88a2093bdd242ae785?auth=3528ca6db1fa99648dfd97179c95e8ac#477">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589123">test_wallet_add_remove_watch_only_account</a>, id: 727232 </b></td></tr><tr><td><p><blockquote><div>Device 1: `Text` is `0x8d2413447ff297d30bdc475f6d5cb00254685aae` </div><div>Device 1: Click system back button </div></blockquote></p>

```
critical/test_wallet.py:256: in test_wallet_add_remove_watch_only_account
    self.home_view.driver.fail(
base_test_case.py:178: in fail
    pytest.fail('Device %s: %s' % (self.number, text))
 Device 1: Incorrect address '0x8d2413447ff297d30bdc475f6d5cb00254685aae' is shown when swiping between accounts, expected one is '0x8d2413447ff297d30bdc475f6d5cb00254685aae' 
```

<b>[[Missing networks in account address, https://github.com/status-im/status-mobile/issues/20166]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8a1f294a06cc40cb8ce3f500791c784f?auth=a7060ec5afc844ba25cd6dd96ce7504b#102">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589114">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><p><blockquote><div># STEP: Change device time so chat will be unmuted by timer</div><div>Device 2: Long press on `ChatElement` </div></blockquote></p>

```
critical/chats/test_group_chat.py:497: in test_group_chat_mute_chat
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Chat is still muted after timeout 
```

<b>[[Chat is not unmuted after expected time: https://github.com/status-im/status-mobile/issues/19627]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a11648612a284f9a96765fd850e232d2?auth=ed908297bd5b75fa53d7f757acdc54bb#477">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2feba5de55d548ee84bd4e9114e1ebba?auth=72657322f17d5bf5578aac5daeac06aa#275">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/778ce659ad644b0e9746ad2fe7d24e14?auth=d03e8f5cb8530505d3fddab6fb6dde5b#242">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (41)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589118">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e9b63d6084d2489d88e7990839a48d20?auth=0e8e7254ece447af7bf8ac25c8c7c47e#52">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589079">test_links_deep_links</a>, id: 702775 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e9b63d6084d2489d88e7990839a48d20?auth=0e8e7254ece447af7bf8ac25c8c7c47e#203">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589083">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a7f234d18f7640a78c8eb4af1ab954f9?auth=1d8139975171ef9620a0c1d9fa05efe2#180">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2fbbc5ee5afe4db8ab9d8bf119b10cab?auth=38e02a9d973ce9886f82a92d0cc207b9#129">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589082">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a7f234d18f7640a78c8eb4af1ab954f9?auth=1d8139975171ef9620a0c1d9fa05efe2#99">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2fbbc5ee5afe4db8ab9d8bf119b10cab?auth=38e02a9d973ce9886f82a92d0cc207b9#84">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589115">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a7f234d18f7640a78c8eb4af1ab954f9?auth=1d8139975171ef9620a0c1d9fa05efe2#114">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2fbbc5ee5afe4db8ab9d8bf119b10cab?auth=38e02a9d973ce9886f82a92d0cc207b9#116">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589084">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2d2ecfd2b73a4342992a52a86aadc689?auth=7d89aadb57d19221d8d8039486c09803#139">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cc7cb86ff7534f88a2093bdd242ae785?auth=3528ca6db1fa99648dfd97179c95e8ac#109">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589095">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2d2ecfd2b73a4342992a52a86aadc689?auth=7d89aadb57d19221d8d8039486c09803#304">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cc7cb86ff7534f88a2093bdd242ae785?auth=3528ca6db1fa99648dfd97179c95e8ac#273">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589081">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cdbb6d6c3609490eb1117a767ce0e76f?auth=5299a6a832699894694bd0eaaacd24f9#155">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9e45460023044d3eb38c4d0e0d8f4c42?auth=d80b4e8de3e8c69e6c860d4a361bb905#121">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589076">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cdbb6d6c3609490eb1117a767ce0e76f?auth=5299a6a832699894694bd0eaaacd24f9#566">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9e45460023044d3eb38c4d0e0d8f4c42?auth=d80b4e8de3e8c69e6c860d4a361bb905#411">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589088">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cdbb6d6c3609490eb1117a767ce0e76f?auth=5299a6a832699894694bd0eaaacd24f9#425">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9e45460023044d3eb38c4d0e0d8f4c42?auth=d80b4e8de3e8c69e6c860d4a361bb905#262">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589078">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cdbb6d6c3609490eb1117a767ce0e76f?auth=5299a6a832699894694bd0eaaacd24f9#367">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9e45460023044d3eb38c4d0e0d8f4c42?auth=d80b4e8de3e8c69e6c860d4a361bb905#212">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589099">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cdbb6d6c3609490eb1117a767ce0e76f?auth=5299a6a832699894694bd0eaaacd24f9#474">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9e45460023044d3eb38c4d0e0d8f4c42?auth=d80b4e8de3e8c69e6c860d4a361bb905#293">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589113">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cdbb6d6c3609490eb1117a767ce0e76f?auth=5299a6a832699894694bd0eaaacd24f9#505">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9e45460023044d3eb38c4d0e0d8f4c42?auth=d80b4e8de3e8c69e6c860d4a361bb905#351">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589074">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cdbb6d6c3609490eb1117a767ce0e76f?auth=5299a6a832699894694bd0eaaacd24f9#217">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9e45460023044d3eb38c4d0e0d8f4c42?auth=d80b4e8de3e8c69e6c860d4a361bb905#167">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589073">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cdbb6d6c3609490eb1117a767ce0e76f?auth=5299a6a832699894694bd0eaaacd24f9#99">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9e45460023044d3eb38c4d0e0d8f4c42?auth=d80b4e8de3e8c69e6c860d4a361bb905#84">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589122">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8a1f294a06cc40cb8ce3f500791c784f?auth=a7060ec5afc844ba25cd6dd96ce7504b#38">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589075">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a11648612a284f9a96765fd850e232d2?auth=ed908297bd5b75fa53d7f757acdc54bb#357">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2feba5de55d548ee84bd4e9114e1ebba?auth=72657322f17d5bf5578aac5daeac06aa#232">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/778ce659ad644b0e9746ad2fe7d24e14?auth=d03e8f5cb8530505d3fddab6fb6dde5b#228">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589111">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a11648612a284f9a96765fd850e232d2?auth=ed908297bd5b75fa53d7f757acdc54bb#284">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2feba5de55d548ee84bd4e9114e1ebba?auth=72657322f17d5bf5578aac5daeac06aa#170">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/778ce659ad644b0e9746ad2fe7d24e14?auth=d03e8f5cb8530505d3fddab6fb6dde5b#141">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589110">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a11648612a284f9a96765fd850e232d2?auth=ed908297bd5b75fa53d7f757acdc54bb#173">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2feba5de55d548ee84bd4e9114e1ebba?auth=72657322f17d5bf5578aac5daeac06aa#106">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/778ce659ad644b0e9746ad2fe7d24e14?auth=d03e8f5cb8530505d3fddab6fb6dde5b#86">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589085">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a11648612a284f9a96765fd850e232d2?auth=ed908297bd5b75fa53d7f757acdc54bb#156">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2feba5de55d548ee84bd4e9114e1ebba?auth=72657322f17d5bf5578aac5daeac06aa#78">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/778ce659ad644b0e9746ad2fe7d24e14?auth=d03e8f5cb8530505d3fddab6fb6dde5b#78">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589086">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a11648612a284f9a96765fd850e232d2?auth=ed908297bd5b75fa53d7f757acdc54bb#324">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2feba5de55d548ee84bd4e9114e1ebba?auth=72657322f17d5bf5578aac5daeac06aa#205">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/778ce659ad644b0e9746ad2fe7d24e14?auth=d03e8f5cb8530505d3fddab6fb6dde5b#201">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589098">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5bf06561a73a47189a71f44517b15a60?auth=d7bf57863b4b317779b585539ec26bb1#99">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/53f9cda629c34fc095144736dfd0cb37?auth=fb34da4f1fac575aba0a366a36c1c842#121">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589097">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5bf06561a73a47189a71f44517b15a60?auth=d7bf57863b4b317779b585539ec26bb1#56">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/53f9cda629c34fc095144736dfd0cb37?auth=fb34da4f1fac575aba0a366a36c1c842#55">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589103">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ee8fbd87eeb84c538668b358915452da?auth=21ab56bfdda9439807645911683ff9ff#128">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0241ba1644104c55a51297ed3fe50c9f?auth=4cf10d43fdd6cf5e1544698ffe61355f#98">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589109">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c361512893954eaaa83b5a3543218a4b?auth=cbda886319ff0cbd4d73f8c19de2eb81#254">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cca1dbaedc234739ba0b1b574254b60d?auth=057b0a8b085052065147994da05893d2#157">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589100">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c361512893954eaaa83b5a3543218a4b?auth=cbda886319ff0cbd4d73f8c19de2eb81#279">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cca1dbaedc234739ba0b1b574254b60d?auth=057b0a8b085052065147994da05893d2#218">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589091">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c361512893954eaaa83b5a3543218a4b?auth=cbda886319ff0cbd4d73f8c19de2eb81#327">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cca1dbaedc234739ba0b1b574254b60d?auth=057b0a8b085052065147994da05893d2#278">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589107">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c361512893954eaaa83b5a3543218a4b?auth=cbda886319ff0cbd4d73f8c19de2eb81#573">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cca1dbaedc234739ba0b1b574254b60d?auth=057b0a8b085052065147994da05893d2#466">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589102">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c361512893954eaaa83b5a3543218a4b?auth=cbda886319ff0cbd4d73f8c19de2eb81#446">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cca1dbaedc234739ba0b1b574254b60d?auth=057b0a8b085052065147994da05893d2#386">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589119">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c361512893954eaaa83b5a3543218a4b?auth=cbda886319ff0cbd4d73f8c19de2eb81#608">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cca1dbaedc234739ba0b1b574254b60d?auth=057b0a8b085052065147994da05893d2#492">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589089">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c361512893954eaaa83b5a3543218a4b?auth=cbda886319ff0cbd4d73f8c19de2eb81#158">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cca1dbaedc234739ba0b1b574254b60d?auth=057b0a8b085052065147994da05893d2#106">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589094">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c361512893954eaaa83b5a3543218a4b?auth=cbda886319ff0cbd4d73f8c19de2eb81#351">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cca1dbaedc234739ba0b1b574254b60d?auth=057b0a8b085052065147994da05893d2#295">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589093">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c361512893954eaaa83b5a3543218a4b?auth=cbda886319ff0cbd4d73f8c19de2eb81#176">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cca1dbaedc234739ba0b1b574254b60d?auth=057b0a8b085052065147994da05893d2#123">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589092">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c361512893954eaaa83b5a3543218a4b?auth=cbda886319ff0cbd4d73f8c19de2eb81#416">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cca1dbaedc234739ba0b1b574254b60d?auth=057b0a8b085052065147994da05893d2#356">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589105">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/874bef5924674c71ac521eeea96f3464?auth=e19aa8d276fa7939c570684007128da2#128">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/11fedfd585074eaf8b591b767abdcc55?auth=785d9030b75ed837cc8eed1e0430a78a#100">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589106">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/874bef5924674c71ac521eeea96f3464?auth=e19aa8d276fa7939c570684007128da2#169">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/11fedfd585074eaf8b591b767abdcc55?auth=785d9030b75ed837cc8eed1e0430a78a#116">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589108">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/170ec4f93b254a509084ad1016405c43?auth=85e9d732532d57b73c433edc12138290#229">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589077">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/170ec4f93b254a509084ad1016405c43?auth=85e9d732532d57b73c433edc12138290#76">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589101">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/170ec4f93b254a509084ad1016405c43?auth=85e9d732532d57b73c433edc12138290#105">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589096">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/170ec4f93b254a509084ad1016405c43?auth=85e9d732532d57b73c433edc12138290#52">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3589112">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/170ec4f93b254a509084ad1016405c43?auth=85e9d732532d57b73c433edc12138290#124">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

qoqobolo commented:
Thanks for your work @yqrashawn, everything looks good to me, but I have one question:

is the checkbox/button color in the scope of this PR?
Currently, they don't inherit the custom color. WDYT, should it be fixed here? 
 
<img width="308" alt="Screenshot 2024-06-28 at 14 06 56" src="https://github.com/status-im/status-mobile/assets/67952253/7872db18-b20f-4a22-9093-3c0d3a42d0c4">

qoqobolo commented:
@yqrashawn Alisher has just updated the design (now VPN is mentioned in the copy). Discussed [here](https://github.com/status-im/status-mobile/issues/20525#issuecomment-2196906128) and [here](https://discord.com/channels/624634427930312714/928625369542713396/1256238342966743152) 

Figma: https://www.figma.com/design/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?node-id=12958-36180&t=Lk6h5r7RglW1XPcc-4

Would you mind updating the copy, please?

yqrashawn commented:
> copy change

I've updated the copy, thanks for the link. @qoqobolo 

> customization color

Fixed the one in settings -> syncing (only one after login)
![CleanShot 2024-07-01 at 09 49 07](https://github.com/status-im/status-mobile/assets/15090582/e7c6f256-5723-4937-8110-8bae4503ae99)

qoqobolo commented:
Thank you @yqrashawn! LGTM

@Francesca-G the PR is ready for design review, thanks!

Francesca-G reviewed (commented):
Here's the [review](https://www.figma.com/design/oRQ5TcgMHKkK79EUfURg3e/Review-Onboarding-%2F-Mobile?node-id=33-1278&t=m7v5LaNMgmEixvim-1) 🙏

yqrashawn commented:
Thanks for the review @Francesca-G, styles are fixed now


![simulator_screenshot_94E7C589-24C5-4C6D-97BE-26FDEE08B533](https://github.com/status-im/status-mobile/assets/15090582/b7b48d87-18a0-4094-9a9c-550e29a8aaed)
